aiohttp/web_response.py

```diff
@@ -101,22 +118,22 @@ def set_status(self, status, reason=None, _RESPONSES=RESPONSES):
         self._reason = reason
 
     @property
-    def keep_alive(self):
+    def keep_alive(self) -> Optional[bool]:
         return self._keep_alive
 
-    def force_close(self):
+    def force_close(self) -> None:
         self._keep_alive = False
 
     @property
-    def body_length(self):
+    def body_length(self) -> int:
         return self._body_length
 
     @property
-    def output_length(self):
+    def output_length(self) -> None:
         warnings.warn('output_length is deprecated', DeprecationWarning)
-        return self._payload_writer.buffer_size
+        return self._payload_writer.buffer_size  # type: ignore
 
-    def enable_chunked_encoding(self, chunk_size=None):
+    def enable_chunked_encoding(self, chunk_size: Any=None) -> None:
```

Comment:
`chunk_size: Optional[int]`. 
The parameter will be removed in aiohttp 4.0 but now let's be consistent.

Reply:
I figure that since it is not used in anyway and setting it to anything else other than `None` will anyway result in an error, we actually don't have to care about that. But if you prefer to have it as `Optional[int]`, let it be so.